tee-ar-ex/trx-python
# Additional script for tractogram

1) Validation script
Uses the header information for checks
- Verify invalid coordinates
- Verify streamlines with 0 or 1 points
- Verify if streamlines perfectly overlap (optional)
- Save tractogram containing only valid streamlines (optional)

2) Concatenate script
Support all file extensions
- Concatenate multiple tractograms together
- Should work if all dpv and dps are the same
- If not, options are present to --delete_dpp and --delete_dps to allow concatenation
